Caltech-IPAC/firefly
# IRSA-975: NED searching seems not to work

https://caltech-ipac.atlassian.net/browse/IRSA-975
- column names must be enclosed in double-quotes
- remove caching done at the search processor’s level
- introduce SharedDbProcessor: results from same processor share the same database
- convert IBE’s template into a search processor
- convert XYGenericProcessor into an EmbeddedDbProcessor
- use firefly’s config file where applicable.
- limit log message to 2k characters

To Test:
- build firefly
- Catalogs -> NED -> m16 -> 1.2 deg
Should get a tri-view with decimated chart and image plot.
What caused the problem was the column names.  They contains spaces, parentheses, etc
This ticket should fix all of that.  There is no restricting on column name now.
